airsonic-advanced.json

```
@@ -0,0 +1,67 @@
+{
+    "Airsonic Advanced": {
+        "containers": {
+            "airsonic-advanced": {
+                "image": "linuxserver/airsonic-advanced",
+                "launch_order": 1,
+                "ports": {
+                    "4040": {
+                        "description": "Webserver Port. Required default: 4040.",
+                        "host_default": 4040,
+                        "label": "Webserver port",
+                        "protocol": "tcp",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Airsonic configuration. Eg: create a Share called airsonic-config for this purpose alone.",
+                        "label": "Config Storage",
+                        "min_size": 1073741824
+                    },
+                    "/music": {
+                        "description": "Select the Share containing your Media",
+                        "label": "Music Storage",
+                        "min_size": 1073741824
+                    },
+                    "/playlists": {
+                        "description": "Choose a Share for Airsonic playlists. Eg: create a Share called airsonic-playlists for this purpose alone.",
+                        "label": "Playlist Storage",
+                        "min_size": 1073741824
+                    },
+                    "/podcasts": {
+                        "description": "Select the Share to store podcasts",
+                        "label": "Podcast Storage",
+                        "min_size": 1073741824
+                    }
+                },
+                "environment": {
+                    "MAX_MEM": {
+                        "description": "This environment variable is used to set the maximum Java heap size in megabytes - 150 is a good default",
+                        "label": "MAX_MEM",
+                        "index": 1
+                    },
+                    "PUID": {
+                        "description": "Enter a valid UID of an existing user with permission to media shares to run Airsonic as (default: 1001).",
+                        "label": "UID",
+                        "index": 2,
+                        "default": 1001
+                    },
+                    "PGID": {
+                        "description": "Enter a valid GID of an existing user with permission to media shares to run Airsonic as (default: 100.",
+                        "label": "GID",
+                        "index": 3,
+                        "default": 100
+                    }
+                }
+            }
+        },
+        "description": "Airsonic music server. <p>Based on a linuxserver docker image: <a href='https://hub.docker.com/r/linuxserver/airsonic-advanced' target='_blank'>https://hub.docker.com/r/linuxserver/airsonic-advanced</a>, available for x86-64 and arm64 architectures only.</p>",
```

To keep aligned with the architecture tag, can you change the last line?
```suggestion
        "description": "Airsonic music server. <p>Based on a custom docker image: <a href='https://hub.docker.com/r/linuxserver/airsonic-advanced' target='_blank'>https://hub.docker.com/r/linuxserver/airsonic-advanced</a>, available for amd64 and arm64 architecture.</p>",
```

---

> To keep aligned with the architecture tag, can you change the last line?

Yes I have accordingly changed the description.